### PR TITLE
elaboration to steps of pcn-iptables

### PR DESCRIPTION
**Pull request by acloudiator**
Few elaborations to pcn-iptables document which observed during installation.

### CODE REVIEW

**Comment by mauriciovasquezbernal** [comment]
@acloudiator why not to integrate this into https://github.com/polycube-network/polycube/pull/144 instead of having a new PR that changes a couple of lines?

File: src/components/iptables/README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ make && sudo make install
 
 ```
 # Start polycubed, in other terminal (or background)
-sudo polycubed
+sudo polycubed &
```
Comment (mbertrone) [inline src/components/iptables/README.md]: Another possibility is to launch polycubed in daemon mode.

**Comment by acloudiator** [comment]
> I just had a better look at this PR. I think this is unnecessary, #144 is deleting this readme file and moving everything under Documentation.

Do you recommend to move it under Documentation? Or You find the PR **unnecessary** and drop it??

File: src/components/iptables/README.md (hunk 1 above)
Comment (mauriciovasquezbernal) [reply src/components/iptables/README.md]: This is in fact the best way to do it.
To use polycubed --daemon.

**Comment by mauriciovasquezbernal** [comment]
> > I just had a better look at this PR. I think this is unnecessary, #144 is deleting this readme file and moving everything under Documentation.
> 
> Do you recommend to move it under Documentation? Or You find the PR **unnecessary** and drop it??

I think we can move this into Documentation.

File: src/components/iptables/README.md (hunk 1 above)
Comment (acloudiator) [reply src/components/iptables/README.md]: Wasn't the `d` at the end of polycube resembles `daemon`. I don't know what that `d` is then :)
Certainly, I'll modify this option with the neat option `--daemon`.

Comment (frisso) [reply src/components/iptables/README.md]: @mauriciovasquezbernal I agree with Aasif, I had the same issue in the past. Having 'd' at the end of 'polycubed', I also assumed it runs as a daemon. Personally, I feel more clear to reverse the behaviour, i.e., polycubed runs as daemon, while something else like 'polycubed -foreground' runs in the terminal.

**Comment by mauriciovasquezbernal** [comment]
@acloudiator could you please update this so we can merge this one?

**Pull request merged**